### PR TITLE
Add chats to draw

### LINKED ISSUE #85: Chat In draws

### PR DESCRIPTION
**Pull request by Mariocj89**
- Add js file for the chats
- Add css file for the chats
- Add chat in draw_base if draw is public
- Add methods in mongo driver to retrieve and add messages
- Expose methods through ws in the views
- Add the views in the urls
- Call ws from js using ajax

closes #85

### CODE REVIEW

File: web/urls.py [hunk 1]
```diff
@@ -26,4 +26,6 @@
     url(r'^ws/favourites/add/$', views.add_favorite, name="ws_add_favorite"),
     url(r'^ws/favourites/remove/$', views.remove_favorite, name="ws_remove_favorite"),
     url(r'^ws/check_access_to_draw/$', views.check_access_to_draw, name="check_access_to_draw"),
+    url(r'^ws/chat/add/$', views.add_message_to_chat, name="chat_add_message"),
+    url(r'^ws/chat/get/$', views.get_chat_messages, name="chat_get_message"),
```
Comment (dnaranjo89) [inline web/urls.py]: the name should be "chat_get_messages" (plural)

File: web/templates/draws/draw_base.html [hunk 2]
```diff
@@ -106,6 +107,67 @@
 				shared_type_field.attr('value','Public');
 	        }
 	    });
+
+    {% if not is_public %}
```
Comment (dnaranjo89) [inline web/templates/draws/draw_base.html]: This part won't work when merged with https://github.com/etcaterva/EchaloASuerte/tree/create_public_draw.
I guess you realized when doing it that it doesn't make sense that all of this is rendered when "is_public" == False.

Comment (Mariocj89) [inline web/templates/draws/draw_base.html]: when we do the merge we'll have to take that into acc :)

File: web/templates/draws/draw_base.html [hunk 3]
```diff
@@ -106,6 +107,67 @@
...
+                chat.html("");
+                var arr = data.messages;
+                for (var i = 0, length = arr.length; i < length; i++) {
+                  var element = arr[i];
+                  var entry = formatChatEntry(element);
+                  chat.append(entry);
+                }
+            }
+        });
+        timeout = setTimeout(updateChat, 5000);
```
Comment (dnaranjo89) [inline web/templates/draws/draw_base.html]: So far it's more than enough but have you think about using websockets in future?

Comment (Mariocj89) [inline web/templates/draws/draw_base.html]: yes I do, this is kind of an "just acceptable" solution. Django does not like much websockets, but we should find a better solution, some kind of commet based at least. Nevertheless, I expect this to change together with the future implementation of the notifications of results in a draw.

File: static/js/chat.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+/**
+ * Function that given an chat entry generates
+ *  and returnsthe html code necessarry to be
+ *  printed in the chat
```
Comment (dnaranjo89) [inline static/js/chat.js]: A space is missing. And let's use the verb render instead of print.

**Pull request merged**